i18n/es/docusaurus-plugin-content-docs/version-V3/subgraph.md

```diff
@@ -2,50 +2,50 @@
 sidebar_position: 6
 ---
 
-# Subgraph
+# Subgrafo
 
-[The Graph](https://thegraph.com/) is a protocol for indexing networks like Ethereum and IPFS.
-Site owners publish _subgraphs_ that expose site data for anyone to query.
-Semaphore's subgraph allows you to retrieve data from the [`Semaphore.sol`](https://github.com/semaphore-protocol/semaphore/blob/main/packages/contracts/Semaphore.sol) smart contract.
+[The Graph](https://thegraph.com/) es un protocolo para indexar redes como Ethererum o IPFS. 
+Las personas dueñas de los sitios publica _subgrafos_ que exponen los datos del sitio para que cualquiera los pueda consultar. 
+El subgrafo de Semaphore le permite obtener datos del contrato inteligente [`Semaphore.sol`](https://github.com/semaphore-protocol/semaphore/blob/main/packages/contracts/Semaphore.sol).
 
-:::tip
-The Graph protocol uses the [GraphQL](https://graphql.org/) query lanaguage. For examples, see the [GraphQL API documentation](https://thegraph.com/docs/developer/graphql-api). Visit the [subgraph repository](https://github.com/semaphore-protocol/subgraph) to see the list of Semaphore subgraphs.
+:::consejo
```

Comment:
```sugestion
:::tip
```

@vplasencia I think we need to keep these keywords in English

Reply:
Yes, right